app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,5 @@
+<div data-go-back-link='true' class='go-back-link' style='display:none;'>
+  <%= link_to 'javascript:;', class: "#{content_column_class} #{label_column_offset_class}", data: { no_sunet: true, target: '[data-no-sunet-content]', content_target: '[data-no-sunet-target]', link: '[data-go-back-link]' } do %>
```

Comment:
I don't think you want the `content_column_class` here.  We really only need the offset and don't necessarily want the link element to take up 9 columns itself.  This makes the underline really far beneath the link and extend the full 9 columns.

![bootstrap-column](https://cloud.githubusercontent.com/assets/96776/7467767/75c5abb8-f2b4-11e4-9126-ca48369abb4b.png)